clients/web/test/testUtils.js

```diff
@@ -670,6 +658,32 @@ girderTest.importStylesheet = function (css) {
 };
 
 /**
+ * Import the JS and CSS files for a plugin.
+ */
+girderTest.importPlugin = function (pluginName) {
+    girderTest.addScript('/static/built/plugins/' + pluginName + '/plugin.min.js');
+    girderTest.importStylesheet('/static/built/plugins/' + pluginName + '/plugin.min.css');
```

Comment:
Yeah, it will just add a new `<link>` tag at the top of the page, pointing to a non-existent URL, which is harmless.